source/contracts/reporting/InitialReporter.sol

```diff
@@ -59,24 +59,17 @@ contract InitialReporter is DelegationTarget, Ownable, BaseReportingParticipant,
         return true;
     }
 
-    function resetReportTimestamp() public returns (bool) {
+    function returnRepFromDisavow() public returns (bool) {
         require(IMarket(msg.sender) == market);
-        if (reportTimestamp == 0) {
-            return;
-        }
-        reportTimestamp = controller.getTimestamp();
+        require(reputationToken.transfer(owner, reputationToken.balanceOf(this)));
```

Comment:
Owner is the original creator of the initialReporter. since we re-use initialReporters, won't this reward the wrong person if say, a market was first migrated to a winning universe, then inside THAT universe, there was a "returnRepFromDisavow" ?

Reply:
owner gets set whenever a report occurs so any legitimate REP in the InitialReport will be associated with the last account to do a report.

If for example this was migrated to a winning universe then either it is not altered at all because there was no report, or the owner is changed to the account that migrates when they do the report. Then if inside of that universe `returnRepFromDisavow` is called the user that migrated and paid for the report will get their REP back.

Reply:
I see. owner remains on the old owner, but it doesn't matter because it is value-less? And the minute a report comes in, and there _IS_ value, then owner is set.